### PR TITLE
sync code from copy destination

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
sync code from copy destination

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Release note**:

```
NONE

```

### CODE REVIEW

File: plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -180,8 +184,10 @@ func isSecretExpired(secret *api.Secret) bool {
 // Copied from kubernetes/cmd/kubeadm/app/util/token
 
 var (
-	tokenRegexpString = "^([a-z0-9]{6})\\.([a-z0-9]{16})$"
-	tokenRegexp       = regexp.MustCompile(tokenRegexpString)
+	// TokenRegexpString defines id.secret regular expression pattern
+	TokenRegexpString = "^([a-z0-9]{6})\\.([a-z0-9]{16})$"
+	// TokenRegexp is a compiled regular expression of TokenRegexpString
+	TokenRegexp = regexp.MustCompile(TokenRegexpString)
```
Comment (liggitt) [inline plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go]: don't export these, we don't really want things starting to depend on this package for these

Comment (zouyee) [reply plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go]: sorry for my mistake, thank you!

**Comment by zouyee** [comment]
ping @liggitt PTAL

**Comment by zouyee** [comment]
gentle ping @liggitt would you like LGTM? thanks！

**Comment by zouyee** [comment]
priority/important-soon

**Comment by thockin** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/56568)

**Comment by zouyee** [comment]
kind/cleanup

**Pull request merged**